shiva/src/test/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaSvCallingTest.scala

```diff
@@ -128,30 +129,39 @@ class ShivaSvCallingTest extends TestNGSuite with Matchers {
     for (del <- bool;
          dup <- bool;
          inv <- bool;
-         tra <- bool) yield Array(1, del, dup, inv, tra)
+         bnd <- bool;
+         ins <- bool) yield Array(1, del, dup, inv, bnd, ins)
   }
 
   @Test(dataProvider = "dellyOptions")
-  def testShivaDelly(bams: Int, del: Boolean, dup: Boolean, inv: Boolean, tra: Boolean): Unit = {
+  def testShivaDelly(bams: Int,
+                     del: Boolean,
+                     dup: Boolean,
+                     inv: Boolean,
+                     bnd: Boolean,
+                     ins: Boolean): Unit = {
     val outputDir = ShivaSvCallingTest.outputDir
     dirs :+= outputDir
 
     val map = Map("sv_callers" -> List("delly"),
                   "delly" ->
-                    Map("DEL" -> del, "DUP" -> dup, "INV" -> inv, "TRA" -> tra))
+                    Map("DEL" -> del, "DUP" -> dup, "INV" -> inv, "BND" -> bnd, "INS" -> ins))
     val pipeline = initPipeline(map, outputDir)
 
     pipeline.inputBams = Map("bam" -> ShivaSvCallingTest.inputTouch("bam" + ".bam"))
 
-    if (!del && !dup && !inv && !tra) intercept[IllegalStateException] {
+    if (!del && !dup && !inv && !bnd && !ins) intercept[IllegalStateException] {
       pipeline.init()
       pipeline.script()
     } else {
       pipeline.init()
       pipeline.script()
 
-      pipeline.functions.count(_.isInstanceOf[DellyCaller]) shouldBe
-        ((if (del) 1 else 0) + (if (dup) 1 else 0) + (if (inv) 1 else 0) + (if (tra) 1 else 0))
+      pipeline.functions.count(_.isInstanceOf[DellyCallerCall]) shouldBe
+        ((if (del) 1 else 0) + (if (dup) 1 else 0) + (if (inv) 1 else 0) + (if (bnd) 1 else 0) + (if (ins)
+                                                                                                    1
+                                                                                                  else
+                                                                                                    0))
```

Comment:
This a bit ugly auto formatted. This has todo with the max line length. If you rewrite is like this:
``` scala
pipeline.functions.count(_.isInstanceOf[DellyCallerCall]) shouldBe
  ((if (del) 1 else 0) + (if (dup) 1 else 0) + (if (inv) 1 else 0) +
    (if (bnd) 1 else 0) + (if (ins)1 else 0))
```

This way the auto formater does need to use this nasty tricks ;)